src/libraries/System.Runtime/tests/TrimmingTests/InvariantGlobalizationTrue.cs

```diff
@@ -30,26 +28,29 @@ static int Main(string[] args)
             return -3;
         }
 
-        foreach (MemberInfo member in globalizationMode.GetMembers(allStatics))
+        if (OperatingSystem.IsWindows())
```

Comment:
We should verify that the GlobalizationMode type is being trimmed on non-Windows. That's the point of this test - to ensure that the right things are being trimmed when InvariantGlobalization is set to `true`.

Reply:
Yeah I wouldn't expect us needing to fix trimming tests as part of this. If the tests are failing now, then we likely have a product bug instead, so we should probably disable the test and log an issue to fix the product bug.